docs/en/stack/ml/rules.asciidoc

```diff
@@ -0,0 +1,39 @@
+[role="xpack"]
+[[ml-rules]]
+=== Machine learning rules and filters
+++++
+<titleabbrev>Rules and filters</titleabbrev>
+++++
+
+By default, as described in <<ml-analyzing>>, anomaly detection is unsupervised 
+and the {ml} models have no awareness of the domain of your data. As a result, 
+{ml} jobs might identify events that are statistically significant but are 
+uninteresting when you know the larger context. Machine learning rules enable 
+you to customize anomaly detection. 
+
+_Rules_ instruct anomaly detectors to change their behavior based on 
+domain-specific knowledge that you provide. When you create a rule, you can  
+specify conditions, scope, and actions. By default, the scope of a rule is 
+unlimited and you must specify at least one condition. When the conditions of a 
```

Comment:
This might be a bit misleading as if scope is specified there could be no conditions.